sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -662,6 +662,14 @@ case class ShowCurrentNamespace(catalogManager: CatalogManager) extends LeafComm
     AttributeReference("namespace", StringType, nullable = false)())
 }
 
+/**
+ * The logical plan of the SHOW CATALOGS command.
+ */
+case class ShowCatalogs(catalogManager: CatalogManager) extends LeafCommand {
+  override val output: Seq[Attribute] = Seq(
+    AttributeReference("catalog", StringType, nullable = false)())
```

Comment:
do we have a reference? what other databases return in SHOW CATALOGS?

Reply:
just show `name` in presto.

Reply:
can you link to the presto doc? and can you also check some traditional RDBMS?

Reply:
in big data：
1.[Presto](https://prestodb.io/docs/current/sql/show-catalogs.html),  There is no output information in the link, but actual output is just `name` 
2.[Flink](https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/sql/show/#run-a-show-statement).
in RDBMS
The catalog layer is not strictly implemented in RDS
1.In PG, Catalog(=database).Schema(namespace).Table, So `show catalogs` == `show databases`; Unfortunately, pg doesn't have this command. we should use `select statement` instead of `show` command. databases; [difference-between-a-catalog-and-a-schema-in-a-relational-database](https://stackoverflow.com/questions/7022755/whats-the-difference-between-a-catalog-and-a-schema-in-a-relational-database)
2.In MySQL, There is no catalog, just schema(database).table. So we can create a database use create schema/database xxx
3.In Oracle,The is no catalog, if we want access metadata, we can use select statement.[Link](https://en.wikipedia.org/wiki/Oracle_metadata )

Reply:
MySQL 8 support catalogs.

> 1. add syntax SHOW CATALOGS [LIKE wild]
   a) add function int mysql_show_catalogs(THD *thd, const char *wild);
   b) add SQLCOM_SHOW_CATALOGS
   c) add processing of SQLCOM_SHOW_CATALOGS to mysql_execute_command
      to use mysql_show_catalogs
   d) add new syntax in sql_yacc.yy, lex.h

https://dev.mysql.com/worklog/task/?id=942

Reply:
@pan3793 Thank you for input. But I'm pretty sure it doesn't support catalog in MySQL any version. The link you gave is only a work log. There is no implementation for it. In MySQL:
database == schema == catalog == a namespace within the server. And all command entry point in server is do_command in sql_parse.cc, there is no case for catalog.

Reply:
Yea, just saw the task status is Un-Assigned